bert/README.md

```diff
@@ -0,0 +1,244 @@
+
+```bash
+composer main.py yamls/mosaic-bert-base-uncased.yaml
+```
+
+**Note:** The `yamls/*.yaml` files are intended to be used with `main.py`.
+
+**Please remember** to modify the reference YAMLs (e.g., `yamls/mosiac-bert-base-uncased.yaml`) to customize saving and loading locations -- only the YAMLs in `yamls/test-cpu/` are ready to use out-of-the-box.
+
+## GLUE fine-tuning
+
+The GLUE benchmark measures the average performance across 8 NLP classification tasks (again, here we exclude the WNLI task). This performance is typically used to evaluate the quality of the pre-training: once you have a set of weights from your MLM task, you fine-tune those weights separately for each task and then compute the average performance across the tasks, with higher averages indicating higher pre-training quality.
+
+To handle this complicated fine-tuning pipeline, we provide the `glue.py` script. which handles parallelizing each of these fine-tuning jobs across all the GPUs on your node.
+To clarify, the `glue.py` script takes advantage of the small dataset/batch size of the GLUE tasks through *task* parallelism (where different tasks are trained in parallel, each using its own GPU), rather than the typical data parallelism (where one task is trained at a time and training batches are parallelized across GPUs).
+
+Once you have modified the YAMLs in `yamls/glue/` to reference your pre-trained checkpoint as the GLUE starting point, use non-default hyperparameters, etc., run the `glue.py` script using the standard `python` launcher (we don't use the `composer` launcher here because `glue.py` does its own multi-process orchestration):
+
+```bash
+python glue.py yamls/glue/mosaic-bert-base-uncased.yaml
+```
+
+Aggregate GLUE scores will be printed out at the end of the script and can also be tracked using Weights and Biases, if enabled via the YAML.
+Any of the other [composer supported loggers](https://docs.mosaicml.com/en/latest/trainer/logging.html#available-loggers) can be added easily as well!
+
+**Note:** The `yamls/glue/*.yaml` files are intended to be used with `glue.py`.
+
+**Please remember** to modify the reference YAMLs (e.g., `yamls/glue/mosiac-bert-base-uncased.yaml`) to customize saving and loading locations -- only the YAMLs in `yamls/test-cpu/` are ready to use out-of-the-box.
+
+### Running on the MosaicML Cloud
+
+If you have configured a compute cluster to work with the MosaicML Cloud, you can use the `mcloud_run.yaml` reference YAMLs for examples of how to run pre-training and fine-tuning remotely!
+
+Once you have filled in the missing YAML fields (and made any other modifications you want), you can launch pre-training by simply running:
+
+```bash
+mcli run -f yamls/mcloud_run.yaml
+```
+
+Similarly, for GLUE fine-tuning just fill in the missing YAML fields (e.g., to use the pre-training checkpoint as the starting point) and run:
```

Comment:
Also, can we enumerate which fields are missing + split this into multiple sentences?

```suggestion
Similarly, for GLUE fine-tuning, just fill in the missing YAML fields (e.g., to use the pre-training checkpoint as the starting point) and run:
```